MAVProxy/tools/mavflightview.py

```diff
@@ -665,7 +665,7 @@ def __init__(self):
     try:
         import faulthandler, signal
         faulthandler.register(signal.SIGUSR1)
-    except ImportError:
+    except Exception as e:
```

Comment:
This is way too generic an error to be catching.

What's the error you get in the original code?

Reply:
... I'll also note that we use `ImportError` in many other places so if it is a problem it's a big problem :-)

Reply:
the problem is that is you haven't install faulthandler in system when you import that package, you will have importError, that's right, but if you had install faulthandler but this module doesn't support register function in windows, they will have another error.
pls reference: https://docs.python.org/3/library/faulthandler.html

Reply:
What is that other error?  My guess is another exception type - we can add it to the list of Exceptions caught - rather than just catching the base class exception and heedlessly going on.

Reply:
fixed